library/src/types.ts

```diff
@@ -242,10 +242,15 @@ export interface MessageTrait {
   description?: DescriptionHTML;
   externalDocs?: ExternalDocs;
   deprecated?: boolean;
-  examples?: any[];
+  examples?: Example[];
   protocolInfo?: Record<string, any>;
 }
 
+export interface Example {
+  headers?: any;
```

Comment:
`object` type, not any :P

Reply:
it was a first change I did when working on this PR, was not yet feeling comfortable with something more than `any` 😄